src/build.sh

```diff
@@ -352,6 +402,10 @@ function parse_build_options()
       --)
         shift
         ;;
+      TEST_MODE)
+        options_values['TEST_MODE']='TEST_MODE'
+        shift
+        ;;
```

Comment:
I don't think we need this.

Reply:
We really don't need that, I ended up making a mistake.